### PR TITLE
Door::ExactState/IsMoving/IsFully(Open/Close)

### PR DESCRIPTION
**Pull request by louis1706**

### CODE REVIEW

File: Exiled.API/Features/Door.cs [hunk 1]
```diff
@@ -89,6 +89,26 @@ public Door(DoorVariant door, Room room)
         /// </summary>
         public Room Room { get; }
 
+        /// <summary>
+        /// Gets a value indicating whether or not the door is fully close.
```
Comment (Michal78900) [inline Exiled.API/Features/Door.cs:93]: Gets a value indicating whether or not the door is fully ~~close~~ **closed**.

File: Exiled.API/Features/Door.cs [hunk 2]
```diff
@@ -89,6 +89,26 @@ public Door(DoorVariant door, Room room)
...
+        /// </summary>
+        public bool IsFullyClose => ExactState is 0;
```
Comment (Michal78900) [inline Exiled.API/Features/Door.cs:95]: ~~IsFullyClose~~ **IsFullyClosed**

File: Exiled.API/Features/Door.cs [hunk 3]
```diff
@@ -89,6 +89,26 @@ public Door(DoorVariant door, Room room)
...
+
+        /// <summary>
+        /// Gets a value indicating whether or not the door is fully open.
+        /// </summary>
+        public bool IsFullyOpen => ExactState is 1;
+
+        /// <summary>
+        /// Gets a value indicating whether or not the door is in movement.
```
Comment (Michal78900) [inline Exiled.API/Features/Door.cs:103]: I would change `the door is in movement` to just `the door is moving`

**Review by Michal78900 — CHANGES_REQUESTED** [review]
Every `or not` in `whether or not the something` is redundant.

File: Exiled.API/Features/Door.cs [hunk 4]
```diff
@@ -90,7 +90,27 @@ public Door(DoorVariant door, Room room)
...
+
+        /// <summary>
+        /// Gets a value indicating whether the door is fully open.
+        /// </summary>
+        public bool IsFullyOpen => ExactState is 1;
+
+        /// <summary>
+        /// Gets a value indicating whether or not the door is open.
+        /// </summary>
+        public bool IsMoving => ExactState is not(0 or 1);
```
Comment (IRacle1) [inline Exiled.API/Features/Door.cs:105]: `The door is open`? Maybe `the door is moving`?

Comment (louis1706) [reply Exiled.API/Features/Door.cs:105]: Oups fuck

**Review by Thundermaker300 — APPROVED** [review]

**Review by Jesus-QC — APPROVED** [review]

**Pull request merged**